FiveCalls/FiveCalls/MyImpactViewController.swift

```diff
@@ -156,6 +156,9 @@ class MyImpactViewController : UITableViewController {
             // Fetch the user's call stats
             let userStatsOp = FetchUserStatsOperation()
             userStatsOp.completionBlock = {
+                if let error = userStatsOp.error {
+                    Answers.logCustomEvent(withName: "userstats-error", customAttributes: ["message": error.localizedDescription])
```

Comment:
Can we use Mixpanel hear too?

Reply:
We can - though I feel like we're misusing either tool using it like this. Is there a generic error logging tool that would do a better job for it?